### PR TITLE
feat: add account info UI

### LINKED ISSUE #7968: [Task]: Add account info UI
### Task description

Add the info of selected account in the account management. Only UI, the logic is done in another issue
Captura de pantalla de 2024-02-01 19-27-40

### PR DESCRIPTION
**Pull request by evavirseda**
## Summary

> Closes #7968 

...

## Changelog

```
- Please write a list of changes
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/views/dashboard/account-management/AccountManagement.svelte [hunk 1]
```diff
@@ -1,48 +1,12 @@
...
-        }
-        return truncateString(address, 7, 5)
-    }
-
-    function handleAccountClick() {
-        // TODO: Implement account details
-    }
+    const implicitAccounts: OutputData[] = $selectedWallet.implicitAccountOutputs
+    const accounts: OutputData[] = $selectedWallet.accountOutputs
+    const allAccounts: OutputData[] = [...accounts, ...implicitAccounts]
```
Comment (cpl121) [inline packages/desktop/views/dashboard/account-management/AccountManagement.svelte:9]: outdated

File: packages/desktop/components/AccountManagementList.svelte [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    function isAnImplicitAccount(output: OutputData) {
+        return implicitAccounts.find((account) => account.outputId === output.outputId)
+    }
+
+    function formatAndTruncateAccount(output) {
+        let address: string
+        if ((output as AccountOutput).accountId) {
+            address = getBech32AddressFromAddressTypes(new AccountAddress((output as AccountOutput).accountId))
+        } else {
+            address = getBech32AddressFromAddressTypes(
```
Comment (cpl121) [inline packages/desktop/components/AccountManagementList.svelte:25]: these changes are outdated respect to the base branch

**Review by cpl121 — CHANGES_REQUESTED** [review]
2 minimal comments 🙏🏼

**Review by cpl121 — APPROVED** [review]
LGTM

**Review by marc2332 — APPROVED** [review]

**Pull request merged**